ArgumentValidator.php

```diff
@@ -89,6 +89,10 @@ private function validateDefinitionArgument($className, Definition $definition)
 
     private function validateExpressionArgument($className, Expression $expression, $allowsNull)
     {
+        if (!$this->isExpressionLanguageSupported()) {
+            return;
+        }
```

Comment:
@matthiasnoback Given that people might still install "invalid" dependencies (if they decide to stick with 2.6.1), it's probably better to just skip the validation of expressions instead of just ignoring this part in tests. This way, service configurations would be validated as usual except for expressions (but they won't be able to use them at all). That's also the reason why I added an extra build for DependencyInjection component in version 2.6 with the ExpressionLanguage component in version 2.5. What do you think?